lib/handlers.js

```diff
@@ -75,6 +75,7 @@ exports.put = function(req, res, next) {
 
   // Update in 1 atomic operation on the database if not specified otherwise
   if (this.shouldUseAtomicUpdate) {
+    this.update_options.new = true;
```

Comment:
Can we move this so that the examples set this? If mongoose and mongodb are moving to use the other default, we shouldn't have different behavior baked in. This also prevents users from overriding it.

You'll have to call

`Resource.updateOptions({ new: true })`